homeassistant/components/tilt_pi/sensor.py

```diff
@@ -0,0 +1,96 @@
+"""Support for Tilt Hydrometer sensors."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .coordinator import TiltPiConfigEntry, TiltPiDataUpdateCoordinator
+from .entity import TiltEntity
+from .model import TiltHydrometerData
+
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+ATTR_TEMPERATURE = "temperature"
+ATTR_GRAVITY = "gravity"
+
+
+@dataclass(frozen=True, kw_only=True)
+class TiltEntityDescription(SensorEntityDescription):
+    """Describes TiltHydrometerData sensor entity."""
+
+    value_fn: Callable[[TiltHydrometerData], StateType]
+
+
+SENSOR_TYPES: Final[list[TiltEntityDescription]] = [
+    TiltEntityDescription(
+        key=ATTR_TEMPERATURE,
+        name="Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda data: data.temperature,
+    ),
+    TiltEntityDescription(
+        key=ATTR_GRAVITY,
+        name="Gravity",
+        native_unit_of_measurement="SG",
+        icon="mdi:water",
```

Comment:
please use translations for this

Reply:
Done in https://github.com/home-assistant/core/pull/139726/commits/a86ea49e9289c97af35f137de5474ff5dc08e13e